torch/_torch_docs.py

```diff
@@ -1482,6 +1482,134 @@ def merge_dicts(*dicts):
     silently incorrect.
 """)
 
+add_docstr(torch.hsplit,
+           r"""
+hsplit(input, indices_or_sections) -> List of Tensors
+
+Splits :attr:`input`, a tensor with one or more dimensions, into multiple tensors
+horizontally (column wise) according to :attr:`indices_or_sections`. Each split
```

Comment:
Instead of `horizontally (column wise)` let's say `"horizontally"`.

What's a little tricky about these operations is that intuitions about the split being performed become challenging if input has more than 3 dimensions